plugins/modules/monitoring/newrelic_deployment.py

```diff
@@ -131,12 +105,15 @@ def main():
         module.exit_json(changed=True)
 
     # Send the data to NewRelic
-    url = "https://rpm.newrelic.com/deployments.xml"
-    data = urlencode(params)
+    url = "https://api.newrelic.com/v2/applications/%s/deployments.json" % module.params["application_id"]
```

Comment:
I think the string should be percent-encoded (`...json" % quote(module.params["application_id"], safe='')`) (with `from ansible.module_utils.six.moves.urllib.parse import quote`).